pkg/cli/debug_merge_logs.go

```diff
@@ -559,9 +575,16 @@ func seekToFirstAfterFrom(f *os.File, from time.Time, editMode log.EditSensitive
 		if _, err := f.Seek(int64(i), io.SeekStart); err != nil {
```

Comment:
This part is sort of bogus now -- and also `N^2`. If you're going to read all of the data from the file, do it earlier and then do all the searching in memory.

In that world, you should set `i` as the offset in the search. At the end this would look like:

```
// Delete the `fi` and `size` stuff above
data, err := ioutil.ReadAll(f)
if err != nil {
    return err
}
offset := sort.Search(len(data), func(i int) bool {
    decoder, err := log.NewEntryDecoderf(data, editMode, format)
    if err != nil {
			panic(err)
		}
		if err = decoder.Decode(&e); err != nil {
			if err == io.EOF {
				return true
			}
			panic(err)
		}
		return e.Time >= from.UnixNano()
})
return data[offset:], nil